chunkset_tpl.h

```diff
@@ -168,7 +177,11 @@ Z_INTERNAL uint8_t* CHUNKMEMSET(uint8_t *out, unsigned dist, unsigned len) {
     return out;
```

Comment:
Something in these inlined functions appears to be emitting several memcpy calls, I'm wonder if it's the lines above.  It is guaranteed to be a sub vector size so it's going to be only a small handful of move instructions.  It also could be happening in the chunkmemset_# functions, though I would expect the compiler would try to optimize those away.